lib/internal/test_runner/coverage.js

```diff
@@ -335,8 +336,14 @@ class TestCoverage {
         newResult.set(url, script);
         continue;
       }
-      const originalLines = this.getLines(url);
       const { data, lineLengths } = sourceMapCache[url];
+      let offset = 0;
+      const executedLines = ArrayPrototypeMap(lineLengths, (length, i) => {
+        const startOffset = offset;
+        const coverageLine = new CoverageLine(i + 1, StringPrototypeRepeat('.', length), startOffset);
```

Comment:
It looks like the `CoverageLine` constructor does a regex operation on the source, which in this case is just a string of `.`s. Does something need to change in that code? Maybe we don't even need to pass a string here now, just a length?